src/coprocessor/aggregate.rs

```diff
@@ -14,9 +14,8 @@
 use std::cmp::Ordering;
```

Comment:
I would like to keep all the code in `coprocessor` unchanged in this PR.

The old aggregate code should not enter executor directory IMO. It's not one of the new DAG executor and should be removed some time later.